src/apps/components/AppFrame/useUpdateAppToken.ts

```diff
@@ -1,13 +1,12 @@
-// @ts-strict-ignore
 import { createAppsDebug } from "@dashboard/apps/apps-debug";
 import { DashboardEventFactory, Events } from "@saleor/app-sdk/app-bridge";
 import { useEffect, useRef } from "react";
 
 /**
  * https://usehooks.com/usePrevious/
  */
-function usePreviousValue(value) {
-  const ref = useRef();
+function usePreviousValue(value: unknown) {
+  const ref = useRef<unknown>();
```

Comment:
suggestion: I've moved this hook into hooks dir, you could use generic to ease type this hook